src/svg-quirks-mode/svg-renderer.js

```diff
@@ -73,6 +73,13 @@ class SvgRenderer {
     }
 
     /**
+     * @return {[number,number]} the natural size, in Scratch units, of this SVG.
+     */
+    get size () {
```

Comment:
I'd be OK with `size`, `scratchSize`, `measurements`, or maybe `logicalSize` (a generic version of `scratchSize`). I think I'd prefer sticking with `size` for brevity and let the comment explain why it's there & what it means.